crates/services/consensus_module/poa/src/verifier.rs

```diff
@@ -31,28 +31,21 @@ pub trait Database {
     fn block_header_merkle_root(&self, height: &BlockHeight) -> StorageResult<Bytes32>;
 }
 
+// TODO: Make this function `async` and await the synchronization with the relayer.
 pub fn verify_consensus(
```

Comment:
should we rename this to `verify_poa_consensus`?

Reply:
It lives inside of `PoA`, so we don't need to add this prefix. We need to rename `verify_poa_block_fields` into `verify_block_fields`